WalletWasabi.Fluent/Controls/Dialog.axaml.cs

```diff
@@ -216,13 +225,25 @@ protected override void OnPropertyChanged<T>(AvaloniaPropertyChangedEventArgs<T>
 
 		if (change.Property == IsDialogOpenProperty)
 		{
-			PseudoClasses.Set(":open", change.NewValue.GetValueOrDefault<bool>());
+			var isOpen = change.NewValue.GetValueOrDefault<bool>();
+
+			PseudoClasses.Set(":open", isOpen);
+
+			if (!isOpen)
+			{
+				PseudoClasses.Set(":alert", false);
+			}
```

Comment:
Is this necessary? 

In the `Close()` method, it is set to false so `:alert` would be set to false anyway.

Reply:
IsDialogOpen property is bound to view model so we need to handle can when property changed.

Reply:
Yes

https://github.com/zkSNACKs/WalletWasabi/blob/a62e10e552d9a392ece5c3aaff275ce50fa08579/WalletWasabi.Fluent/ViewModels/Dialogs/Base/DialogScreenViewModel.cs#L32

https://github.com/zkSNACKs/WalletWasabi/blob/a62e10e552d9a392ece5c3aaff275ce50fa08579/WalletWasabi.Fluent/ViewModels/Dialogs/Base/DialogScreenViewModel.cs#L45